### PR TITLE
💀 Deathdate … and anniversary! 🎉

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

**Comment by jancborchardt** [comment]
_Not sure if I should be worried about you or me now …_

File: js/services/vCardProperties.js [hunk 1]
```diff
@@ -66,6 +66,14 @@ angular.module('contactsApp')
 			readableName: t('contacts', 'Birthday'),
 			template: 'date'
 		},
+		anniversary: {
+			readableName: t('contacts', 'Anniversary'),
+			template: 'date'
+		},
+		deathdate: {
+			readableName: t('contacts', 'Date of Death'),
```
Comment (jancborchardt) [inline js/services/vCardProperties.js:74]: Correct capitalization should be »Date of death«, only first character capitalized.

**Comment by DeepDiver1975** [comment]
> Not sure if I should be worried about you or me now …

no worry this introduces anniversary as well ....

File: js/services/vCardProperties.js (hunk 1 above)
Comment (DeepDiver1975) [inline js/services/vCardProperties.js:74]: please fix right away .... thx

**Comment by DeepDiver1975** [comment]
sync to calendar is in core - https://github.com/owncloud/core/pull/25655

**Comment by jancborchardt** [comment]
So @ChristophWurst noted something: Does it really make sense to save date of death for a person in Contacts? Cause … you will not contact them anymore really.

I guess it makes sense to remember someone though, or just for completeness … I just wanna make sure we’re not just stupidly implementing the spec, because well, this in particular is a very slim usecase. ;)

**Comment by spoorun** [comment]
@jancborchardt A good point. We had thought of that, and from a personal perspective, I retain contacts who have died, often for a memory or commemoration - for many people it is an important a date to remember someone - as important as a birthday is of someone still alive. So it's not a huge use case, but certainly not a 'very slim' one. As people grow older, alas that usecase also grows.

**Comment by jancborchardt** [comment]
Ok, just wanted to make sure it’s a conscious decision adding this, not just cause you’re in the process of adding anniversary anyway. :)

**Comment by MariusBluem** [comment]
Our satanistic users will like that :smiling_imp:

**Comment by spoorun** [comment]
I guess there's nothing preventing future events being recorded either.. for those who are exceptionally well-planned!

**Pull request merged**